addOns/llm/src/test/java/org/zaproxy/addon/llm/services/LlmCommunicationServiceUnitTest.java

```diff
@@ -62,4 +78,50 @@ void shouldBeConsideredReviewdIfMarkedAsSuch() {
         // Then
         assertThat(result, is(equalTo(true)));
     }
+
+    private static final Confidence CONFIDENCE =
+            new Confidence(Alert.CONFIDENCE_MEDIUM, "explanation");
```

Comment:
Better keep constants at the top of the class.

Reply:
Tweaked, was just an artifact from rebase.